scripts/install/autobrr.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# autobrr installer
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_reqs=("curl" "sqlite3")
+apt_install "${app_reqs[@]}"
+
+users=($(_get_user_list))
+
+if [[ -n $1 ]]; then
+    user=$1
+    _autobrr_user_config ${user}
+    if [[ -f /install/.nginx.lock ]]; then
+        echo_progress_start "Configuring nginx"
+        bash /etc/swizzin/scripts/nginx/autobrr.sh
+        systemctl reload nginx
+        echo_progress_done
+    fi
```

Comment:
The config function does not start the service it seems
```suggestion
    fi
    systemctl enable -q --now autobrr@${user} 2>&1 | tee -a $log
```